.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,25 +7,36 @@ assignees: ''
 
 ---
 
-NOTE: If this is a _discussion starter_, you need any _installation help_ or have a _question_ on how to accomplish something,
+<!--
+NOTE: If this is a _discussion starter_, you need any _installation help, or have a _question_ on how to accomplish something,
 rather post at [our discussion channel](https://www.domjudge.org/chat) or send an email to our
 [DOMjudge-devel mailinglist](https://www.domjudge.org/mailman/postorius/lists/domjudge-devel.domjudge.org/).
 
 Thank you for suggesting ways to improve DOMjudge. Before you file a feature
 request, it might be useful to discuss it first via the chat or mailing list
 linked above. We can then assess together whether there is
 not already a way to accomplish your goal with DOMjudge currently.
+-->
 
 ### Description of the enhancement request
-> Replace this line with a short description.
+<!--
+Replace this line with a short description.
+-->
 
 ### The goal you want to achieve
-> Please elaborate on the (larger, higher level) goal you want to achieve with this enhancement, so we have a good understanding what this feature would be useful for and how it fits in DOMjudge as a whole.
+<!--
+Please elaborate on the (larger, higher level) goal you want to achieve with this enhancement, so we have a good understanding what this feature would be useful for and how it fits in DOMjudge as a whole.
+-->
 
 ### Expected behaviour
-> Replace this line with what you would expect to happen.
-> Please describe the workflow how you want this feature to work with expected start URL
```

Comment:
This sentence has been poorly formulated.

Reply:
I've updated the wording.